Server/setup.sh

```diff
@@ -2,8 +2,17 @@
 # this script will need to be run as sudo
 
 curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
+echo "Installing NodeJS"
 apt-get install -y nodejs
 
+echo "Installing build-essential"
 apt-get install -y build-essential
 
+echo "Installing MongoDB"
+apt-get install -y mongodb
+
+echo "Installing pm2"
 npm install -g pm2
+
+echo "Installing Dependencies"
+npm install
```

Comment:
One thing I did notice is that setup.sh is stored in the ./Server/. Unless you think they're running sudo ./Server/setup.sh. Either way, I'll make the change.